### ISSUE TITLE
Please update to 1.20

### ISSUE THREAD

**Issue by LordFeathers** [Virtuoel/NoFog#26]
### What is your suggestion?

Please update to 1.20. This mod is a lifesaver. Thanks!

**Comment by Virtuoel** [comment]
Duplicate of #25.

The 1.3.4 release is 1.20.x compatible, and is out on Modrinth. The CurseForge upload is waiting on the project page owner to upload the new files and then on CurseForge to approve those files.

**Comment by LordFeathers** [comment]
Thanks.
